tendrl/gluster_integration/objects/node_context/__init__.py

```diff
@@ -37,7 +37,7 @@ def _create_node_id(self, node_id=None):
         with open(local_node_context, 'wb+') as f:
```

Comment:
A cluster will have multiple nodes, which context are you going to save to /clusters? We are already saving ClusterNodeContext which is done by the node-agent.

Please remove this object